addons/stock_account/models/product.py

```diff
@@ -172,6 +182,10 @@ def _change_standard_price(self, new_price, counterpart_account_id=False):
         :param new_price: new standard price
         """
         # Handle stock valuation layers.
+
+        if self.valuation == 'real_time' and not self.env['account.move'].check_access_rights('create', raise_exception=False):
+            raise UserError(_('Cannot create change the price manually if the valuation mode is automatic and if you do not have the right to create accounting entries.'))
```

Comment:
to reword

Reply:
"An accounting entry has to be created and you don’t have the mandatory access rights."

Reply:
apparently oma encountered another one

oma@odoo.com: Sorry, you are not allowed to access documents of type 'Stock Valuation Layer' (stock.valuation.layer). This operation is allowed for the groups:      - Inventory/Administrator - (Operation: read, User: 6)

Reply:
"You cannot update the cost of a product in automated valuation as it leads to the creation of a journal entry, for which you don't have the access rights."